bpf/include/bpf_common.h

```diff
@@ -53,7 +53,7 @@ struct {
     __uint(map_flags, BPF_F_NO_PREALLOC);
     __type(key, int);
     __type(value, struct sock_storage_data);
-} map_of_sock_storage SEC(".maps");
+} km_map_of_sock_storage SEC(".maps");
```

Comment:
More than 16 characters will be truncated. `km_sock_storage` ?

Reply:
Done